app/_hub/kong-inc/key-auth/how-to/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+nav_title: Configure realms for regional Consumers
+title: Configure realms for regional Consumers in {{site.konnect_short_name}}
+---
+
+Starting in {{site.base_gateway]] 3.10.x, credentials that are stored centrally in {{site.konnect_short_name}} that are shared across multiple {{site.base_gateway}} can be validated by configuring `identity_realms` field in the Key Auth plugin. You can use these realms when you create [regional Consumers](/konnect/regional-consumers/).
```

Comment:
Do we want to add some language that says "today a DP can only reach out to realms in the same region as they are deployed"?